docs/git-workflow.md

```diff
@@ -29,4 +29,8 @@ Keeping Your Branch Up To Date
 
 While it is tempting to merge from `master` into your branch frequently, this leads to a messy history because each merge creates a merge commit. When working by yourself, it is best to use `git pull --rebase master`, but if you're pushing to a shared repo, it is best to not do any merging or rebasing until the feature is ready for final testing, and then do a [rebase](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) at the very end. This is one reason why it is important to open pull requests whenever you have working code.
 
-If you have a Pull Request branch that cannot be merged into `master` due to a conflict (this can happen for long-running Pull Request discussions), it's still best to rebase the branch (rather than merge) and resolve any conflicts on your local copy before updating the Pull Request with `git push --force`. **Be aware** that this will **replace** any commits currently in your shared branch, so anyone who is also using that branch will be in trouble. Only use `git push --force` if the Pull Request is ready to merge and no one else is using it (or if you have coordinated the force-push with the other developers working on the branch).
+If you have a Pull Request branch that cannot be merged into `master` due to a conflict (this can happen for long-running Pull Request discussions), it's still best to rebase the branch (rather than merge) and resolve any conflicts on your local copy. 
+
+Once you have resolved any conflicts locally you can update the Pull Request with `git push --force-with-lease` (note: using `--force-with-lease` is preferred to `--force` in order to [help to protect remote commits](https://developer.atlassian.com/blog/2015/04/force-with-lease/)). 
```

Comment:
Maybe we could use the active voice and strip out "in order to"? `note: we prefer using --force-with-lease over --force to help protect remote commits`.

Reply:
Fixed